### PR TITLE
Fix docker tag for latest

### PR DESCRIPTION
**Pull request by onyiny-ang**

### CODE REVIEW

File: scripts/imagebuild.sh [hunk 1]
```diff
@@ -54,6 +54,8 @@ echo "Pushing image with tag '${TAG}'."
 docker push ${REGISTRY}${REPO}/federation-v2:${TAG}
 
 if ! [ -z "${LATEST}" ]; then
```
Comment (pmorie) [inline scripts/imagebuild.sh]: since `nounsent` is set, I think you're going to want to say `${LATEST:""}` here

File: scripts/imagebuild.sh [hunk 2]
```diff
@@ -54,6 +54,8 @@ echo "Pushing image with tag '${TAG}'."
...
+
+   docker tag ${REGISTRY}${REPO}/federation-v2:${LATEST}
```
Comment (marun) [inline scripts/imagebuild.sh]: This doesn't look correct - a source is needed as well:

```
docker tag ${REGISTRY}${REPO}/federation-v2:${LATEST} ${REGISTRY}${REPO}/federation-v2:${TAG}
```

Reference: https://docs.docker.com/engine/reference/commandline/tag/

**Comment by irfanurrehman** [comment]
@pmorie @onyiny-ang merging this PR is not important right now, marking https://quay.io/kubernetes-multicluster/federation-v2:v0.0.1 as latest is. I guess that can be done manually this time.

File: scripts/imagebuild.sh (hunk 2 above)
Comment (font) [reply scripts/imagebuild.sh]: Yep. You'll need to put the source image first:

```
docker tag ${REGISTRY}${REPO}/federation-v2:${TAG} ${REGISTRY}${REPO}/federation-v2:${LATEST}
```

File: scripts/imagebuild.sh [hunk 3]
```diff
@@ -53,7 +54,9 @@ docker build ${dockerfile_dir} -t ${REGISTRY}${REPO}/federation-v2:${TAG}
 echo "Pushing image with tag '${TAG}'."
 docker push ${REGISTRY}${REPO}/federation-v2:${TAG}
 
-if ! [ -z "${LATEST}" ]; then
+if [ "$LATEST" = "latest" ]; then
```
Comment (pmorie) [inline scripts/imagebuild.sh]: The `=` operator isn't how the rest of the equality comparison tests in this file are written - would you use `==` so they're consistent?

**Pull request merged**